esy.json

```diff
@@ -60,7 +60,15 @@
     "install-githooks": "./githooks/install.ml",
     "clean-githooks": "./githooks/clean.ml",
     "clean": "dune clean",
-    "watch": "dune build @install --promote-install-files --watch"
+    "watch": "dune build @install --promote-install-files --watch",
+    "odoc": "sh -c \"esy x dune build @doc && ./scripts/sync_odoc.sh > /dev/null\"",
+    "odoc:watch:build": "dune build @doc --watch",
+    "odoc:watch:sync": "./scripts/watch_odoc.sh > /dev/null",
+    "odoc:watch": "concurrently -p none \"esy odoc:watch:build\" \"esy odoc:watch:sync\"",
+    "sphinx": "sh -c \"sphinx-build sphinx _docs/sphinx/ \"",
```

Comment:
Why is the `sh-c` needed here if you're not using `&&`?

Reply:
It isn't :P 
3b05c801d9b171b5d36eb847d3e09754f7be3816